src/app/app.module.ts

```diff
@@ -59,7 +59,7 @@ import { AppState, InternalStateType } from './app.service';
 // Footer & Header
 import { FeatureBannerComponent } from './feature-flag/banner/feature-banner.component';
 import { FeatureFlagResolver } from './feature-flag/resolver/feature-flag.resolver';
-import { FeatureTogglesService } from './feature-flag/service/feature-toggles.service';
+import { FABRIC8_FEATURE_TOGGLES_API_URL, featureTogglesApiUrlProvider, FeatureTogglesService } from './feature-flag/service/feature-toggles.service';
```

Comment:
As far as I can tell FABRIC8_FEATURE_TOGGLES_API_URL isn't used anywhere in this file so it doesn't need to be imported. Am I missing something?